editor/test/resources/test_project/app_manifest/physics_2d_box2dv3.appmanifest

```diff
@@ -1,31 +1,31 @@
 platforms:
     armv7-ios:
         context:
-            excludeLibs: [physics, box2d_defold, script_box2d_defold]
+            excludeLibs: [physics, LinearMath, BulletDynamics, BulletCollision, box2d_defold, script_box2d_defold]
```

Comment:
There was an issue with this manifest: it didn't exclude `LinearMath`, `BulletDynamics`, and `BulletCollision` libs (parts of 3d physics)